java/configuration/src/main/java/sleeper/configuration/properties/table/TableProperty.java

```diff
@@ -326,7 +326,7 @@ public interface TableProperty extends SleeperProperty {
     TableProperty INGEST_BATCHER_INGEST_MODE = Index.propertyBuilder("sleeper.table.ingest.batcher.ingest.mode")
```

Comment:
Now `BatchIngestMode` no longer exists, maybe this can be called `INGEST_BATCHER_INGEST_QUEUE` or INGEST_BATCHER_DESTINATION_QUEUE` or something similar?

Reply:
Done

Reply:
The property key also needs updating here, the templates would need regenerating too

Reply:
That's a breaking change, but I've done that now.